### ISSUE TITLE
Robustness in the same range of color temperature

### ISSUE THREAD

**Issue by wxy1234567** [huawei-noah/multi_hyp_cc#16]
Thank you for your excellent work!

After the experiment, I found that the algorithm in my own data will exist over fitting problem. Training data and validation data are extracted from some videos and then labeled. The validation set is very good, but the performance is poor in scenes that never appear (the same color temperature range and no mixed light source, and the test set label is within the kmeans clustering range). Of course, the model can be improved by adding a few similar scenes.

 I try to reduce the network and use some data enhancement, But it only has a alleviating effect. I want to have good robustness for every scene in a specific color temperature range.

Do you have any good suggestions?

**Comment by dhernandez0** [comment]
Thanks for your interest! My suggestions would be:
1. Have at least 100 candidate illuminants (see Table 7 in arxiv version)
2. Make sure all videos are captured by the same camera and same processing steps (demosaicing etc)
3. Use with multi-device training: The main strength of our work is the ability to use different datasets (captured with different cameras) to train. So, I would try training on all public datasets available + your dataset to boost results.
